plugins/janus_audiobridge.c

```diff
@@ -1256,6 +1279,21 @@ void janus_audiobridge_destroy_session(janus_plugin_session *handle, int *error)
 	return;
 }
 
+static void janus_audiobridge_notify_participants(janus_audiobridge_participant *participant, json_t *msg) {
+	/* participant->room->participants_mutex has to be locked. */
```

Comment:
Yes , in videoroom I used this one:
`janus_audiobridge_notify_participants`
but in audiobridge there was no such helper but on every place notification code part was done separately, so  Idon't know if this will be ok